assets/js/marker.js

```diff
@@ -407,15 +412,8 @@ class Marker {
       .end()
       .find('img.background').attr('src', bgUrl);
   }
-  updateOpacity(opacity = Settings.markerOpacity, isInvisibleRemovedMarkers = Settings.isInvisibleRemovedMarkers) {
-    let targetOpacity;
-    if (this.canCollect) {
-      targetOpacity = opacity;
-    } else {
-      targetOpacity = isInvisibleRemovedMarkers ? 0 : opacity / 3;
-    }
-
-    this.lMarker && this.lMarker.setOpacity(targetOpacity);
+  updateOpacity(opacity = Settings.markerOpacity) {

```

Comment:
It's simply reverting back to [the previous version](https://github.com/jeanropke/RDR2CollectorsMap/commit/7733fdd501e48e6116654cc171ff7a114d2e962a#diff-bf0d6c5df235063d74b2c94a8537d36db1370df5e9e874c3d3136169e20e0f59L410-L411).